barretenberg/cpp/scripts/_benchmark_remote_lock.sh

```diff
@@ -9,6 +9,13 @@ function cleanup() {
 }
 
 # Check for existing lock file
+for i in {1..10} ; do
+    if ! ssh $BB_SSH_KEY $BB_SSH_INSTANCE "test -f ~/BENCHMARK_IN_PROGRESS"; then
```

Comment:
snuck in this, some waiting logic so you can start multiple benchmarks and walk away